etc/ci/performance/test_perf.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+wget http://people.mozilla.org/~jmaher/taloszips/zips/tp5n.zip -O tp5n.zip && \
+unzip -o tp5n.zip
+
+virtualenv venv --python=/usr/bin/python3
```

Comment:
Do you mean ```virtualenv venv --python=/usr/bin/env python3```?
It does not work the same way.
I guess ```/usr/bin/env virtualenv venv --python=python3``` is what you want.
But, I have not seen anyone use this.